k9mail/src/main/java/com/fsck/k9/ui/settings/general/GeneralSettingsDataStore.kt

```diff
@@ -76,7 +76,7 @@ class GeneralSettingsDataStore(
             "messageview_return_to_list" -> K9.setMessageViewReturnToList(value)
             "messageview_show_next" -> K9.setMessageViewShowNext(value)
             "quiet_time_enabled" -> K9.setQuietTimeEnabled(value)
-            "disable_notifications_during_quiet_time" -> K9.setNotificationDuringQuietTimeEnabled(value)
```

Comment:
Same here. Change to
```kotlin
"disable_notifications_during_quiet_time" -> K9.setNotificationDuringQuietTimeEnabled(!value)
```

Reply:
I thought about doing this, but I think it'll be also confusing.
What about changing the variable name to something like: "shouldDisableNotificationsDuringQuietTime"?